strings/charoccurrence/charoccurrence_test.go

```diff
@@ -0,0 +1,40 @@
+package charoccurrence
```

Comment:
This should be tested outside the package like so
```suggestion
package charoccurrence_test
```
Although you should move this to the strings package rather than create its own package.

Reply:
- [x] fixed